### PR TITLE
Issue #357: <ul> common default padding and margin given for all the …

### PR DESCRIPTION
**Pull request by omernadeem1070**
…unordered lists.

## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

Default padding was added to (ul) element with the bullet points and the event times. Bullet points are left-aligned with the rest of the content.

Before
After

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -21,48 +18,23 @@
...
-                  :name="game.name"
-                  :p0ready="game.p0Ready ? 1 : 0"
-                  :p1ready="game.p1Ready ? 1 : 0"
-                  :game-id="game.id"
-                  :status="game.status"
-                  :num-players="game.numPlayers"
-                  :is-ranked="game.isRanked"
-                />
+                <game-list-item :name="game.name" :p0ready="game.p0Ready ? 1 : 0" :p1ready="game.p1Ready ? 1 : 0"
+                  :game-id="game.id" :status="game.status" :num-players="game.numPlayers" :is-ranked="game.isRanked" />
```
Comment (itsalaidbacklife) [inline src/views/HomeView.vue:22]: We generally prefer one-line-per attribute for template code when the tag has many attributes. Please roll this back to the previous multi-line setup. I assume that this was done automatically by the linter/IDE? We clearly could use an improvement to the lint setup to make this more consistent. I think that warrants a separate issue, but for now, please keep the multiple-lines for these longer tags.

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -21,48 +18,23 @@
...
-              color="secondary"
-              class="mt-4"
-              href="https://human-ai-interaction.github.io/cuttle-bot/"
-              target="_blank"
-              data-cy="ai-link"
-              :size="buttonSize"
-            >
+            <v-btn variant="outlined" color="secondary" class="mt-4"
+              href="https://human-ai-interaction.github.io/cuttle-bot/" target="_blank" data-cy="ai-link"
+              :size="buttonSize">
```
Comment (itsalaidbacklife) [inline src/views/HomeView.vue:33]: Same here

File: src/views/HomeView.vue [hunk 3]
```diff
@@ -21,48 +18,23 @@
...
             </v-btn>
-            <v-btn
-              variant="outlined"
-              class="mt-4"
-              href="https://discord.gg/9vrAZ8xGyh"
-              target="_blank"
-              :size="buttonSize"
-            >
+            <v-btn variant="outlined" class="mt-4" href="https://discord.gg/9vrAZ8xGyh" target="_blank"
+              :size="buttonSize">
```
Comment (itsalaidbacklife) [inline src/views/HomeView.vue:37]: and here

File: src/views/HomeView.vue [hunk 4]
```diff
@@ -80,13 +52,8 @@
...
     </v-container>
-    <BaseSnackbar
-      v-model="showSnackBar"
-      :message="snackBarMessage"
-      color="error"
-      data-cy="newgame-snackbar"
-      @clear="clearSnackBar"
-    />
+    <BaseSnackbar v-model="showSnackBar" :message="snackBarMessage" color="error" data-cy="newgame-snackbar"
+      @clear="clearSnackBar" />
```
Comment (itsalaidbacklife) [inline src/views/HomeView.vue:56]: And here

**Review by itsalaidbacklife — APPROVED** [review]
The fix looks good. Requesting two changes:

1. There are a few code formatting changes to roll back
2. Please bump the `version` in package.json a patch-version to 4.1.2

Can you please also comment in this PR with a before and after pic of the change for the patch notes?

File: src/views/HomeView.vue [hunk 5]
```diff
@@ -170,6 +137,10 @@ p {
   margin-top: 1rem;
 }
 
+ul {
+  padding-inline-start: 40px;
+}
```
[the remark this answers is not in the document]
Comment (omernadeem1070) [reply src/views/HomeView.vue:171]: Before and After of the website section right?

File: src/views/HomeView.vue (hunk 1 above)
Comment (omernadeem1070) [reply src/views/HomeView.vue]: Got it.

File: src/views/HomeView.vue (hunk 5 above)
[the remark this answers is not in the document]
Comment (itsalaidbacklife) [reply src/views/HomeView.vue:171]: Yes exactly just to see what the change looks like in the UI of the HomeView page

[the remark this answers is not in the document]
Comment (omernadeem1070) [reply src/views/HomeView.vue:171]: Done.

**Comment by itsalaidbacklife** [comment]
Perfect, thank you. Last change: Can you please bump the `version` field in package.json to 4.1.2? We use labels `major-version`, `minor-version`, and `patch-version` on PRs to flag which one to update and we try to be diligent about incrementing the version with each merge so that we can track which changes are live when

**Review by itsalaidbacklife — APPROVED** [review]
Great work, thanks for you contribution! This resolves #357. I will merge as soon as CI passes

**Comment by omernadeem1070** [comment]
Awesome thank you.

**Pull request merged**